force-app/main/default/lwc/serviceScheduleReview/serviceScheduleReview.html

```diff
@@ -0,0 +1,131 @@
+<template>
+    <c-section title={reviewLabel}></c-section>
+
+    <lightning-layout multiple-rows="true" class="slds-var-p-around_small">
+        <lightning-layout-item
+            size="12"
+            class="slds-text-heading_small 
+            slds-border_bottom 
+            slds-var-p-bottom_x-small"
+        >
+            {schedule.record.Name}
+        </lightning-layout-item>
+
+        <!-- Column 1 -->
+        <lightning-layout-item size="6">
+            <!-- date range -->
+            <c-output-field label="Dates">
+                <lightning-icon
+                    icon-name="utility:event"
+                    alternative-text=""
+                    size="x-small"
+                    class="slds-p-right_x-small"
+                >
+                </lightning-icon>
+                <lightning-formatted-date-time
```

Comment:
Do we need a TODO to test this in different time zones? I'm assuming we are respecting the browser not Salesforce locale here?

Reply:
formatted-date-time interprets the UTC just like a record page does, so we shouldn't need to do anything.

Reply:
Right I think I already asked that, thanks for the reminder!